### PR TITLE
Illustrator consistency

### PR DESCRIPTION
**Pull request by plural**
tidies up illustrator fields.  Standardizes on comma-space separated for multiple credits.

### CODE REVIEW

File: pack/df.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "faction_code": "anarch",
     "faction_cost": 1,
     "flavor": "With each passing day alone, Hoshiko found it harder to think. With each hour, the static grew louder.",
-    "illustrator": "Photo Tammy Gann/Unsplash. Deep Dream",
+    "illustrator": "Tammy Gann/Unsplash, Deep Dream",
```
Comment (NoahTheDuke) [inline pack/df.json:9]: Maybe reach out to Nisei creative for this one. The others make sense as either FFG or Nisei worked with both parties, but these deep dream ones might have different rules about attribution.

Comment (plural) [reply pack/df.json]: yeah, i asked.  Folks weren't clear, so i'm changing this to be treated as a single illustrator.

**Review by NoahTheDuke — APPROVED** [review]

**Review by distributive — APPROVED** [review]
Do we want to consider having the illustrator field as a list rather than a comma-separated string or is that over complicating things?

**Comment by plural** [comment]
yes to changing this to a list.  I'll do that separately, though.

**Pull request merged**